.github/workflows/nightly.yml

```diff
@@ -0,0 +1,61 @@
+name: DJL Demo Nightly test
+
+on:
+  schedule:
+    - cron:  '0 3 * * *'
+  workflow_dispatch:
+
+
+jobs:
+  compile-test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - name: Set up JDK 11
+        uses: actions/setup-java@v1
+        with:
+          java-version: 11
+      - name: Compile Apache Beam
+        run: cd apache-beam/ctr-prediction && mvn package
+      - name: Compile Apache Kafka
+        run: cd apache-kafka && ./gradlew build
+      - name: Compile AWS projects
+        run: |
+          cd aws/aws-kinesis-video-streams && ./gradlew build && cd ..
```

Comment:
Can you use step.working-directory instead of all these cd commands? It would make it a bit easier to manage the jobs?